src/routes/about/+page.md

```diff
@@ -9,22 +9,7 @@
 
 # About
 
-UCANs were originally created by [Brooklyn Zelenka](https://twitter.com/expede) in 2019 as part of her work in creating a set of distributed data storage, communications and distributed computing protocols at [Fission Internet Corp](https://fission.codes).
-
-## Links
-
-* [UCANs working group Github organization](https://github.com/ucan-wg/)
-
-## More Reading
-
- * <OutboundLink href="https://fission.codes/blog/auth-without-backend/">UCAN: Authorizing Users Without a Back End</OutboundLink>
- * <OutboundLink href="https://fission.codes/blog/lightweight-credentials-ucan/">Lightweight Credentials for Offers with UCAN</OutboundLink>
- * <OutboundLink href="https://whitepaper.fission.codes/access-control/ucan">UCANs in the Fission Whitepaper</OutboundLink>
- * The <OutboundLink href="https://www.npmjs.com/package/ucans/">ucans library</OutboundLink> on NPM
-
-## Partners
-
-Coming soon!
+UCANs were originally created by [Brooklyn Zelenka](https://octodon.social/@expede) in 2019 as part of her work in creating a set of distributed data storage, authentication, and distributed computing protocols at [Fission](https://fission.codes).
```

Comment:
```suggestion
UCANs were originally created by [Brooklyn Zelenka](https://octodon.social/@expede) in 2019 as part of her work in creating a set of distributed data storage, authorization, and distributed computing protocols at [Fission](https://fission.codes).
```

We might want to say `authorization` instead of `authentication` because UCAN is an authorization scheme.

UCANs